ejb3/src/main/java/org/jboss/as/ejb3/component/interceptors/LoggingInterceptor.java

```diff
@@ -64,6 +65,9 @@ public Object processInvocation(InterceptorContext interceptorContext) throws Ex
         try {
             // we just pass on the control and do our work only when an exception occurs
             return interceptorContext.proceed();
+        } catch ( EJBComponentUnavailableException ex) {
+            EjbLogger.EJB3_INVOCATION_LOGGER.warn(ex);
```

Comment:
@kabir currently the exception get logged by the following line of code:

```java
EjbLogger.EJB3_INVOCATION_LOGGER.invocationFailed(component.getComponentName(), invokedMethod, t);
```

*(you can see in the JIRA that the message from 'invocationFailed' is wrapped around the one about the container being suspensed)*

So to address the issue as agreed on the JIRA, we need to have the message being log as WARN but *only* when the EJBComponentUnavailableException (all other case should be proceed at usual). Thus the extra catch and the logging under WARN of this one exception.

Now, why do we still throw an exception ? To allow other interceptor to process the error - to ensure the transaction manager still handles properly the end of the transaction (if any). At first, I've tried removing the exception but it led (not surpringsingly) to the unit relating to EJB suspend to be failing. 

As the exception has already been logged in the logger interceptor, I don't expect other interceptor (or other part of the code) to print the stacktrace. But maybe I am wrong on this ? I did not find any occurences of the message being printed during running the test however. 

You probably know the subsystem better than me, so if you think this message will be printed somewhere else, let me know, I'll look into it.